### PR TITLE
Restore "Use Regular Expression"

### PR DESCRIPTION
**Pull request by warpech**
Problem: When PR #197 and #190 were merged, the checkbox Use Regular Expression was lost during merge

Solution: Restore the checkbox Use Regular Expression

### CODE REVIEW

**Comment by PatrykSzwer** [comment]
Good catch! I also committed my proposition of better arrangement of route options. Please take a look and if You find It more readable same as me, please merge.

File: src/WebsiteEditor/wwwroot/WebsiteEditor/viewmodels/RoutingPage.html [hunk 1]
```diff
@@ -60,8 +60,11 @@
                     </div>
                 </div>
                 <div>
-                    <label for="websiteRouteDisabled">Disabled</label>
-                    <input type="checkbox" id="websiteRouteDisabled" checked="{{route.Disabled$::change}}"/>
+                    <label for="websiteCatchingRuleIsRegex">Use Regular Expression</label>
```
Comment (warpech) [inline src/WebsiteEditor/wwwroot/WebsiteEditor/viewmodels/RoutingPage.html]: Can you pls change "regular expression" to be lowercase to be consistent with other labels?

Comment (PatrykSzwer) [reply src/WebsiteEditor/wwwroot/WebsiteEditor/viewmodels/RoutingPage.html]: Done

**Comment by warpech** [comment]
Lgtm

**Pull request merged**